static/js/factories/index.js

```diff
@@ -0,0 +1,105 @@
+// @flow
```

Comment:
what do you think about calling this `factories/dashboard.js` or similar? Just thinking we might eventually want to have separate modules for profiles and so on.